api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TPTCompletionCohortQueries.java

```diff
@@ -312,13 +316,13 @@ public CohortDefinition getTxCurrWithTPTInLast7Months() {
     compositionCohortDefinition.addParameter(new Parameter("location", "Location", Location.class));
 
     String generalParameterMapping =
-        "onOrAfter=${endDate-210d},onOrBefore=${endDate},location=${location}";
+        "onOrAfter=${endDate-7m},onOrBefore=${endDate},location=${location}";
```

Comment:
Here the BA team is talking of 210 days as opposed to exact 7 months, can you reconcile those with Eude and see where to adjust? seems like the values match when months is used?